.eslintrc.js

```diff
@@ -1,20 +1,60 @@
-/**@type {import('eslint').Linter.Config} */
-// eslint-disable-next-line no-undef
-module.exports = {
-  // root: true,
-  // parser: '@typescript-eslint/parser',
-  // plugins: [
-  // 	'@typescript-eslint',
-  // ],
-  // extends: [
-  // 	'eslint:recommended',
-  // 	'plugin:@typescript-eslint/recommended',
-  // ],
-  // rules: {
-  // 	'semi': [2, "always"],
-  // 	'@typescript-eslint/no-unused-vars': 0,
-  // 	'@typescript-eslint/no-explicit-any': 0,
-  // 	'@typescript-eslint/explicit-module-boundary-types': 0,
-  // 	'@typescript-eslint/no-non-null-assertion': 0,
-  // }
+/** @type {import('eslint').Linter.Config} */
+const config = {
+  extends: [
+    'eslint:recommended',
+    'plugin:eslint-comments/recommended',
+    'plugin:@typescript-eslint/recommended',
+  ],
+  env: {
+    browser: true,
+    node: true,
+  },
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    tsconfigRootDir: __dirname,
+    project: [
+      './tsconfig.eslint.json',
+      './apps/**/tsconfig.json',
+      './packages/*/tsconfig.json',
+    ],
+  },
+  plugins: ['@typescript-eslint'],
+  root: true,
+  rules: {
+    'eslint-comments/no-unused-disable': 'error',
+  },
+  overrides: [
+    // override for all typescript files
+    {
+      files: ['**/*.{ts,tsx}'],
+      extends: [
+        // enable type-aware linting rules
+        'plugin:@typescript-eslint/recommended-requiring-type-checking',
+      ],
+    },
+    // override for workspace config/script files
+    {
+      files: ['*.js', 'scripts/*.js'],
+      rules: {
+        // disable `require` errors
+        '@typescript-eslint/no-var-requires': 'off',
+      },
+    },
+    // override for test files
+    {
+      files: ['**/__tests__/**/*.{ts,tsx}', '**/*.{spec,test}.{ts,tsx}'],
+      extends: ['plugin:jest/recommended'],
+      plugins: ['jest'],
+      rules: {
+        '@typescript-eslint/no-empty-function': 'off',
+        '@typescript-eslint/no-non-null-assertion': 'off',
+        '@typescript-eslint/no-unsafe-call': 'warn',
+        '@typescript-eslint/no-unsafe-argument': 'warn',
+        '@typescript-eslint/no-unsafe-assignment': 'warn',
+        '@typescript-eslint/no-unsafe-member-access': 'warn',
+      },
+    },
```

Comment:
This override applies to our test files.

First, we enable [`eslint-plugin-jest`](https://github.com/jest-community/eslint-plugin-jest) plugin and extend from its recommended rules.

Then we disable or change some `@typescript-eslint` errors to warnings because it doesn't necessarily make sense for tests for be entirely type-safe.